cirq-core/cirq/protocols/has_stabilizer_effect_protocol_test.py

```diff
@@ -125,9 +129,20 @@ def test_via_unitary():
     op3 = OpWithUnitary(np.array([[1, 0], [0, np.sqrt(1j)]]))
     assert not cirq.has_stabilizer_effect(op3)
 
+    # 2+ qubit cliffords
```

Comment:
```suggestion
    # 2 qubit cliffords
```

Reply:
it works up to 3 qubit operations like CCNOTs

Reply:
The 3 asserts for cliffords are only for 2 qubit operations, but lgtm